runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -5,17 +5,21 @@ endif::[]
 :page-aliases: general:getting-started:deploy-to-server.adoc, deploying-to-pcf.adoc
 
 
-image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-cloud2-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub 2.0"]
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub 1.0"]
 image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
 image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
 image:logo-rtf-active.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
 
 
 When you run an app in Anypoint Studio, the app deploys to an embedded test server in Studio. Because this server is not meant for production deployment and uptime restrictions apply, deploy your Mule app using one of the deployment options supported by Anypoint Runtime Manager.
 
-* *<<CloudHub Deployments,CloudHub>>*
+* *<<CloudHub 2.0 Deployments,CloudHub 2.0>>*
 +
-Deploy your apps from the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console and host them in xref:cloudhub.adoc[CloudHub].
+Deploy your apps from the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console and host them in xref:hosting::cloudhub-2.adoc[CloudHub 2.0].
+* *<<CloudHub 1.0 Deployments,CloudHub 1.0>>*
++
+Deploy your apps from the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console and host them in xref:cloudhub.adoc[CloudHub 1.0].
 * *<<Hybrid Deployments,Hybrid>>*
 +
 Deploy your apps from the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console and host them on your Mule servers.
```

Comment:
Do you want to say something more - like "on-premises Mule servers"? 
https://docs.mulesoft.com/mule-runtime/4.4/mule-standalone

Reply:
Done